### PR TITLE
Check if device model exist

### PR DESCRIPTION
**Pull request by Dreddik**
Fix for #8

### CODE REVIEW

**Comment by benasher44** [comment]
Looks good! Can you add a spec?

**Comment by Dreddik** [comment]
@benasher44 sorry, i don't know how.

**Comment by benasher44** [comment]
The first step is to figure out what it is in the JSON output from `xcrun simctl list -f devices` causes this bug. Once you have that, you can make a json file with this output in spec/fixtures. And then in spec/unit_spec.rb, you can copy the sample that does something similar (inject the sample json from a file).

**Comment by benasher44** [comment]
This fix should have a CHANGELOG entry as well.

**Comment by Dreddik** [comment]
@benasher44 I added test device, and updated specs. 
I think, it will be better, if you add an entry to changelog.

**Comment by benasher44** [comment]
@Dreddik I can add it, but I'm just curious. What's your concern about modifying the CHANGELOG? You're not expected to do anything about the version or anything like that. You can just add your change to the master section under bug fixes (replacing "None").

**Comment by benasher44** [comment]
Looks good. Nice work! going to wait on CI to pass, and then I think we're all set!

**Comment by Dreddik** [comment]
Rubocop informs about complexity, refactored to avoid.

**Comment by Dreddik** [comment]
God, please, allow this last commit to be passed.

File: spec/find_spec.rb [hunk 1]
```diff
@@ -1,15 +1,22 @@
 require 'fourflusher'
 
 describe Fourflusher::SimControl do
-  let(:simctl) { Fourflusher::SimControl.new }
```
Comment (benasher44) [inline spec/find_spec.rb]: Can we undo the changes in this file? This is an integration spec, and it should be mostly unaffected by your changes.

**Comment by benasher44** [comment]
@Dreddik thanks for your contribution!
